master/buildbot/newsfragments/github_oauth.bugfix

```diff
@@ -0,0 +1 @@
+:class:`GitHubAuth` is not using `Authorization` headers instead of `access_token` query parameter, as the later was deprecated by Github. (:issue:`5188`)
```

Comment:
```suggestion
:class:`GitHubAuth` is now using `Authorization` headers instead of `access_token` query parameter, as the latter was deprecated by Github. (:issue:`5188`)
```